src/view/scripts.inc.twig

```diff
@@ -70,17 +71,18 @@ window.SKOSMOS = {
   {%- endif -%}
 </script>
 
-<!-- Plugin JS sources -->
-{% for files in request.plugins.pluginsJS %}
-  {% for file in files %}<script src="{{ file }}"></script>{% endfor %}
-{% endfor %}
-
 <!-- Bootstrap -->
 <script src="node_modules/bootstrap/dist/js/bootstrap.bundle.js"></script>
 
 <!-- Vue.js -->
 <script src="node_modules/vue/dist/vue.global.js"></script>
 
+<!-- Plugin JS sources -->
+{% for files in request.plugins.pluginsJS %}
+  {% for file in files %}<script src="{{ file }}"></script>{% endfor %}
+{% endfor %}
+{% if request.plugins.callbacks %}<script>var pluginCallbacks = [{% for function in request.plugins.callbacks %}{% if not loop.first %}, {% endif %}"{{ function }}"{% endfor %}];</script>{% endif %}
```

Comment:
This declares a new global JS variable `pluginCallbacks`. Would it make sense to move it within the SKOSMOS object instead, to avoid polluting the global namespace?